aea/aea_builder.py

```diff
@@ -1273,6 +1323,34 @@ def _try_to_load_agent_configuration_file(aea_project_path: Path) -> None:
                 )
             )
 
+    @staticmethod
+    def _check_valid_entrypoint(build_entrypoint: str, directory: str):
+        """
+        Check a configuration has a valid entrypoint.
+
+        :param build_entrypoint: the build entrypoint.
+        :param directory: the directory from where to start reading the script.
+        :return: None
+        """
+        enforce(
+            build_entrypoint is not None,
+            "Package has not a build entrypoint specified.",
+        )
+        build_entrypoint = cast(str, build_entrypoint)
+        script_path = Path(directory) / build_entrypoint
+        enforce(
+            script_path.exists(), f"File '{build_entrypoint}' does not exists.",
+        )
+        enforce(
+            script_path.is_file(), f"'{build_entrypoint}' is not a file.",
+        )
```

Comment:
These are user defined values, so enforce seems not appropriate. We should probably raise an AEAException or similar.

Reply:
in fact, if the condition is not satisfied, `AEAException` is raised with the error message, am I right? How would you improve this code?